gems/smithy/lib/smithy/templates/client/endpoint_provider_spec.erb

```diff
@@ -39,11 +39,11 @@ module <%= namespace %>
 
 <% test_case.operation_inputs.each do |operation_input| -%>
       it 'produces the correct output from the client when calling <%= operation_input.operation_name %>' do
+        Client.add_plugin(stub_send)
```

Comment:
This doesn't look correct - this will break behavior for any other specs trying to use the client

Reply:
Everything currently passes, but I agree, I would like to remove this when stub responses is implemented.